draft-ietf-quic-transport.md

```diff
@@ -2483,8 +2489,8 @@ after a frame bearing the FIN flag is sent.
 
 A stream is "half-closed (remote)" when the stream is no longer being used by
 the peer to send any data.  An endpoint will have either received all data that
-a peer has sent or will have received a RST_STREAM frame and discarded any
-received data.
+a peer has sent, will have sent an UNI flag on the stream, or will have
```

Comment:
Client initiated streams can also be unidirectional as well if mapped out that way, why would a mapping want to specify a mix like that? The PR doesn't have details about how the http mapping changes, would love to see that as well.